packages/cli/src/utils/generate.ts

```diff
@@ -533,6 +535,49 @@ export default function Page(props) {
   }
 }
 
+export async function addPluginsOverrides(basePath: string) {
+  const { userDir, tmpStoreConfigFile, tmpDir } = withBasePath(basePath)
+
+  const { plugins } = (await import(tmpStoreConfigFile)) as {
+    plugins: string[]
+  }
+
+  const indexPluginsOverrides = plugins
+    .filter((plugin) =>
+      existsSync(
+        path.join(userDir, 'node_modules', `${plugin}/dist/components/index.js`)
+      )
+    )
+    .map((plugin) => {
+      const pluginReference =
```

Comment:
Can you add a NOTE here so we can get the name of component in a better way once we decide to go forward?